src/main/java/Nanami.java

```diff
@@ -0,0 +1,289 @@
+        System.out.println("\t|    \\| /     \\  |    \\| /     \\  |    \\/  |  |");
+        System.out.println("\t» ¤ «» ¤ «» ¤ «» ¤ «» ¤ «» ¤ «» ¤ «» ¤ «» ¤ «»\n");
+        System.out.println("» Hello. I am Nanami, your task-manager.\n» What can I do for you?\n");
+    }
+
+    private static void giveFarewell() {
+        // print a farewell to user
+        System.out.println("» Until next time. Goodbye.");
+
+        try {
+            appender = new FileWriter(outfile, true);
+            appender.write("xx");
+        } catch (IOException e) {
+            System.out.println("could not end the file properly");
+        }
+    }
+
+    private static void addToList(String input) {
+        // add user's input to an array of tasks
+        if (input.isEmpty()) {
+            return;
+        }
+        String[] inputArray = input.split("\\s+");
+
+        if (inputArray[0].equals("todo")) {
+            try {
+                ToDo newTask = new ToDo(input.split("/"));
+                tasklist[taskcount++] = newTask;
+                updateFile(1);
+            } catch (EmptyTaskException e) {
+                System.out.println("I can't store a todo like that. It only needs a description, no attachments using /.");
+            } catch (ToDoMismatchedParameterException e) {
+                System.out.println("A todo task does not have any attachments using /. I can't store it like this.");
+            } catch (IOException e) {
+                System.out.println("no save file :(");
+            }
+        } else if (inputArray[0].equals("deadline")) {
+            try {
+                Deadline newTask = new Deadline(input.split("/"));
+                tasklist[taskcount++] = newTask;
+                updateFile(1);
+            } catch (EmptyTaskException e) {
+                System.out.println("The deadline is missing something. I need the task type, description, and /by attachment. Try again.");
+            } catch (DeadlineMismatchedParameterException e) {
+                System.out.println("A deadline has a /by attachment only. I can't store it like this.");
+            } catch (IOException e) {
+                System.out.println("no save file :(");
+            }
+        } else if (inputArray[0].equals("event")) {
+            try {
+                Event newTask = new Event(input.split("/"));
+                tasklist[taskcount++] = newTask;
+                updateFile(1);
+            } catch (EmptyTaskException e) {
+                System.out.println("The event is missing information. I need the task type, description, and /to and /from attachments. Try again.");
+            } catch (EventMismatchedParameterException e) {
+                System.out.println("An event has a /to attachment and a /from attachment only. I can't store it like this.");
+            } catch (IOException e) {
+                System.out.println("no save file :(");
+            }
```

Comment:
avoid long methods! you can consider refactoring this and create new methods for each operations/task